assets/js/mosaic-setup.js

```diff
@@ -183,3 +205,33 @@ function toggleControlPanel() {
 function toggleHighlight() {
     SetHighlight(!highlightOption);
 }
+
+function clockCountdown(first) {
```

Comment:
Not very nice that we have C/P this function both in mosaic-setup.js & setup.js. 

Can we put it in a separate file (e.g. clock.js) and include it in both *.html?